google-cloud-speech/lib/google/cloud/speech/v1/speech_client_config.json

```diff
@@ -1,40 +1,40 @@
 {
   "interfaces": {
-    "google.cloud.speech.v1beta1.Speech": {
+    "google.cloud.speech.v1.Speech": {
       "retry_codes": {
-        "idempotent": [
-          "DEADLINE_EXCEEDED",
-          "UNAVAILABLE"
-        ],
-        "non_idempotent": [
-          "UNAVAILABLE"
-        ]
+          "idempotent": [
+            "DEADLINE_EXCEEDED",
+            "UNAVAILABLE"
+          ],
+          "non_idempotent": [
+            "UNAVAILABLE"
+          ]
       },
       "retry_params": {
         "default": {
           "initial_retry_delay_millis": 100,
           "retry_delay_multiplier": 1.3,
           "max_retry_delay_millis": 60000,
-          "initial_rpc_timeout_millis": 60000,
+          "initial_rpc_timeout_millis": 20000,
           "rpc_timeout_multiplier": 1.0,
-          "max_rpc_timeout_millis": 60000,
+          "max_rpc_timeout_millis": 20000,
           "total_timeout_millis": 600000
         }
       },
       "methods": {
-        "SyncRecognize": {
+        "Recognize": {
           "timeout_millis": 60000,
```

Comment:
@landrito What is the process for updating these configuration values? Can we submit a PR to [googleapis/googleapis](https://github.com/googleapis/googleapis)?